extension/content_script.js

```diff
@@ -251,11 +256,21 @@ function parseDocument() {
 }
 
 let readableDomIds = parseDocument();
+doc = new Doc(readableDomIds);
 tracker = new Tracker(readableDomIds);
-display = new Display(readableDomIds, speed);
+window.tracker = tracker; 
+display = new Display(readableDomIds, speed, doc.getTotalWords());
+window.display = display;
```

Comment:
hm did we revert the change where this is no longer a global var?

Reply:
I believe it is now in master, but this is a relic of a different branch before we fixed it